site-src/blog/2023/0801-welcome-gamma.md

```diff
@@ -0,0 +1,127 @@
+a Service makes it possible to use the Gateway API with a service mesh, but
+there are several interesting use cases that remain difficult to cover.
+
+### How does Gateway API conformance work for a service mesh?
+
+One of the challenges that the GAMMA initiative ran into is that Gateway API
+conformance was strongly tied to the idea that a given implementation provides
+an ingress controller. Many service meshes don't, and requiring a
+GAMMA-conformant mesh to also implement an ingress controller seemed
+impractical at best. This resulted in work restarting on Gateway API
+_conformance profiles_, as discussed in
+[GEP-1709](https://gateway-api.sigs.k8s.io/geps/gep-1709/).
+
+The basic idea of conformance profiles is that we can define subsets of the
+Gateway API, and allow implementations to choose - and document! - which
+subsets they conform to. GAMMA is adding a new profile, named `Mesh`, which
+checks only the mesh functionality as defined by GAMMA; SIG-Network-Policy is
+going to be using this concept as well.
+
+## What else is in Gateway API 0.8.0?
+
+In addition to GAMMA becoming experimental, Gateway API 0.8.0 also includes a
+way to configure routing scope for a given Gateway (see
+[GEP-1651](https://gateway-api.sigs.k8s.io/geps/gep-1651/) and updates the
+HTTPRoute resource to include native configuration for timeouts (see
+[GEP-1742](https://gateway-api.sigs.k8s.io/geps/gep-1742/).
+
+Additionally, we have (by necessity) taken a hard look at some
+[Experimental][status] GEPs which have been lingering long enough that
+projects have come to rely on them in production use. This is a bit of a
+breakdown of the GEP process; in order to prevent it happening in the future,
+we have changed the GEP process such that reaching [Experimental][status]
+_requires_ that a GEP include both the graduation criteria by which the GEP
+will become [Standard][status], and a probationary period after which the GEP
+will be dropped if does not meet its graduation criteria.
+
+For an exhaustive list of changes included in the `v0.8.0` release, please see
+the [v0.8.0 release
+notes](https://github.com/kubernetes-sigs/gateway-api/releases/tag/v0.8.0).
+For more information on Gateway API versioning, refer to the [official
+documentation](https://gateway-api.sigs.k8s.io/concepts/versioning/).
+
+## How can I get started with the Gateway API?
+
+At this point, the Gateway API is supported by a number of
+[implementations][impl] of both ingress controllers and service meshes. To get
+started, take a look at the [API concepts documentation][concepts] and check
+out some of the [Guides][guides] to learn about the Gateway API, or check out
+the [implementations page][impl] and select an implementation that you're
+familiar with to try it out. Gateway API is a [Custom Resource Definition
+(CRD)][crd] based API so you'll need to [install the CRDs][install-crds] onto
+a cluster to use the API.
+
```

Comment:
```suggestion

Below is an example of how an `HTTPRoute` resource now binds to a Kubernetes service to define traffic to multiple backend services. 

```
apiVersion: gateway.networking.k8s.io/v1beta1
kind: HTTPRoute
metadata:
  name: bar-route
spec:
  parentRefs:
    - name: demo-app
       port: 5000
       kind: Service
  rules:
  - matches:
    - headers:
      - type: Exact
        name: env
        value: v1
    backendRefs:
    - name: demo-app-v1
      port: 5000
  - backendRefs:
    - name: demo-app-v2
      port: 5000
```          
          
```

Reply:
Wondering if we could perhaps add an example of how this works?

Reply:
It's a fine example, thanks for the suggestion! I'm including it. 🙂